hangman.py

```diff
@@ -1,34 +1,47 @@
 import random
 from words import words
-from hangman_visual import lives_visual_dict_easy, lives_visual_dict_hard, lives_visual_dict_medium, lives_visual_dict_impossible
+from hangman_visual import lives_visual_dict_easy, lives_visual_dict_hard, lives_visual_dict_medium, \
+    lives_visual_dict_impossible
 import string
 
-
 total_score = 0
 
-
-def get_valid_word(words):
+def get_valid_word(words, lives):
     """Gets word from words.py wihtout '-' or ' '."""
+    word = ""
 
-    word = random.choice(words)
+    if lives == 12:
+        word_dict = [word for word in words if len(word) >= 4 and len(word) <=6] 
```

Comment:
As you can see here the m level still gives words of 6 letters. I don't want that. A minor change needs to be done.

![Screenshot (11)](https://user-images.githubusercontent.com/81291713/115251160-e57f3080-a147-11eb-8c6a-88dfc91ea393.png)

The following code will make it happen:
![Screenshot (12)](https://user-images.githubusercontent.com/81291713/115251407-25deae80-a148-11eb-88be-d085d8a13253.png)


Just make your ```get_valid_word``` look something like this and then the code will be perfectly ready to go.

Reply:
I've updated it now